### DISCUSSION TITLE
How to interpret the metric ‘Number of reads mapped to multiple loci’?

### DISCUSSION THREAD

**Discussion by onmyojiyys** [alexdobin/STAR#2064] [Q&A]
How to interpret the metric ‘Number of reads mapped to multiple loci’? Why is the number obtained by NH> 1 different from the ‘Number of reads mapped to multiple loci’, and how to understand this difference?

**Comment by alexdobin** [comment]
If you take unique read IDs from alignments with NH>1, it should agree with the number of reads mapped to multiple loci.
